### PR TITLE
swiat mumino=kow

### PR DESCRIPTION
**Pull request by UrsPre**
komentarz

### CODE REVIEW

File: muminek.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mala migotka
```
Comment (piotr87) [inline muminek.txt:1]: in english please

Comment (piotr87) [inline muminek.txt:1]: in english please

Comment (UrsPre) [reply muminek.txt:1]: No no no!!!

Comment (piotr87) [reply muminek.txt:1]: asdddasfasdf

Comment (piotr87) [reply muminek.txt:1]: asdfasdfs

Comment (UrsPre) [reply muminek.txt:1]: !!!!

**Review by UrsPre — COMMENTED** [review]
qw

**Comment by UrsPre** [comment]
rfr

**Pull request merged**